tests/unit/test_run.py

```diff
@@ -2295,6 +2295,23 @@ def test_interpolate(combo_df):
     npt.assert_array_almost_equal(res.values.squeeze(), combo.target_values)
 
 
+def test_interpolate_float(scm_run):
+    target_time_points = [float(y) for y in range(2000, 2005)]
+
+    res = scm_run.interpolate(target_time_points)
+    assert res["year"].to_list() == [2000, 2001, 2002, 2003, 2004]
+    assert np.isnan(res.values).sum() == 0
+
+
+def test_interpolate_int(scm_run):
+    # check int not converted to unix time (gh 221)
+    target_time_points = [int(y) for y in range(2000, 2005)]
+
+    res = scm_run.interpolate(target_time_points)
+    assert res["year"].to_list() == [2000, 2001, 2002, 2003, 2004]
```

Comment:
Does this come out as all being 1970 if int is converted to unix time (because it is assumed to be 2000 seconds after 1970, not the year 2000)?

Reply:
Correct

Reply:
Nice, good pick up and fix